mixer/adapter/signalfx/config/config.proto

```diff
@@ -18,10 +18,65 @@ syntax = "proto3";
 // $description: Adapter that sends Istio metrics to SignalFx
 // $location: https://istio.io/docs/reference/config/policy-and-telemetry/adapters/signalfx.html
 
-// The `signalfx` adapter collects Istio metrics and sends them to
-// [SignalFx](https://signalfx.com).
+// The `signalfx` adapter collects Istio metrics and trace spans and sends them
+// to [SignalFx](https://signalfx.com).
 //
-// This adapter supports the [metric template](https://istio.io/docs/reference/config/policy-and-telemetry/templates/metric/).
+// This adapter supports the [metric template](https://istio.io/docs/reference/config/policy-and-telemetry/templates/metric/)
+// and [tracespan template](https://istio.io/docs/reference/config/policy-and-telemetry/templates/tracespan/).
+//
+// If sending trace spans, this adapter can make use of certain conventions in
+// the tracespan format that is configured to send to this adapter.  Here is an
+// example tracespan spec that will work well:
+//
+// ```yaml
+//   apiVersion: config.istio.io/v1alpha2
```

Comment:
Remove extra indent for the YAML content. 'apiVersion' should line up with the first backtick